purr/theme.json

```diff
@@ -0,0 +1,214 @@
+				},
+				{
+					"color": "#aab8c2",
+					"name": "Gray Light",
+					"slug": "custom-gray-light"
+				},
+				{
+					"color": "#14171a",
+					"name": "Different dark blue",
+					"slug": "custom-different-dark-blue"
+				}
+			]
+		},
+		"layout": {
+			"contentSize": "620px",
+			"wideSize": "1000px"
+		},
+		"spacing": {
+			"units": [
+				"%",
+				"px",
+				"em",
+				"rem",
+				"vh",
+				"vw"
+			]
+		},
+		"typography": {
+			"fontFamilies": [
+				{
+					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+					"name": "System Font",
+					"slug": "system-font"
+				}
+			]
+		},
+		"useRootPaddingAwareAlignments": true
+	},
+	"styles": {
+		"blocks": {
+			"core/button": {
+				"color": {
+					"background": "var(--wp--preset--color--custom-twitter-blue)",
+					"text": "var(--wp--preset--color--custom-gray-lightest)"
+				},
+				"elements": {
+					"link": {
+						"color": {
+							"text": "var(--wp--preset--color--custom-gray-lightest)"
+						}
+					}
+				},
+				"typography": {
+					"textTransform": "none"
+				},
+				"variations": {
+					"outline": {
+						"border": {
+							"bottom": {
+								"color": "var(--wp--preset--color--custom-twitter-blue)",
+								"style": "solid"
+							},
+							"left": {
+								"color": "var(--wp--preset--color--custom-twitter-blue)",
+								"style": "solid"
+							},
+							"right": {
+								"color": "var(--wp--preset--color--custom-twitter-blue)",
+								"style": "solid"
+							},
+							"top": {
+								"color": "var(--wp--preset--color--custom-twitter-blue)",
+								"style": "solid"
+							}
+						},
+						"color": {
+							"background": "var(--wp--preset--color--custom-gray-lightest)",
+							"text": "var(--wp--preset--color--custom-dark-blue)"
+						}
+					}
+				}
+			},
+			"core/navigation": {
+				"typography": {
+					"textDecoration": "none"
+				}
+			},
+			"core/navigation-link": {
+				"typography": {
+					"textDecoration": "underline"
+				}
+			},
+			"core/search": {
+				"border": {
+					"bottom": {
+						"style": "none",
+						"width": "0px"
+					},
+					"left": {
+						"style": "none",
+						"width": "0px"
+					},
+					"radius": "0px",
+					"right": {
+						"style": "none",
+						"width": "0px"
+					},
+					"top": {
+						"style": "none",
+						"width": "0px"
+					}
+				},
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--small)",
+					"lineHeight": "2.2"
+				}
+			},
+			"core/separator": {
+				"color": {
+					"background": "var(--wp--preset--color--custom-light-blue)"
+				},
+				"variations": {
+					"wide": {
+						"color": {
+							"background": "var(--wp--preset--color--custom-light-blue)"
+						}
+					}
+				}
+			},
+			"core/site-title": {
+				"typography": {
+					"textDecoration": "none"
+				}
+			}
+		},
+		"color": {
+			"text": "var(--wp--preset--color--custom-dark-blue)"
+		},
+		"css": "",
```

Comment:
what do we need this for? was this generated by CBT?

Reply:
yes!